WalletWasabi/WabiSabi/Backend/Statistics/TxPropagationVerifier.cs

```diff
@@ -0,0 +1,64 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using NBitcoin;
+using WalletWasabi.Logging;
+using WalletWasabi.WabiSabi.Backend.WebClients;
+
+namespace WalletWasabi.WabiSabi.Backend.Statistics;
+
+public class TxPropagationVerifier
+{
+	public TxPropagationVerifier(Network network, IHttpClientFactory httpClientFactory)
+	{
+		Verifiers = new List<ITxPropagationVerifier>()
+		{
+			new BlockstreamApiClient(network, httpClientFactory.CreateClient(nameof(BlockstreamApiClient))),
+			new MempoolSpaceApiClient(network, httpClientFactory.CreateClient(nameof(MempoolSpaceApiClient)))
+		};
+	}
+	public List<ITxPropagationVerifier> Verifiers { get; }
+
+	// Don't throw as this task is not awaited
+	public async Task LogTxAcceptedByThirdPartyAsync(uint256 txid, CancellationToken cancel, int delay = 30000, int attempts = 10)
```

Comment:
No one calls this method with modified `delay` and `attempts`.

Maybe convert those optional parameters to a `private const` field?

Just an idea.